test/test_helper.rb

```diff
@@ -70,6 +62,16 @@
 
 require 'zlib'
 
+ENV['APP_ENV'] ||= 'test'
+require File.expand_path('../../config/environments/' + ENV['APP_ENV'], __FILE__)
+Dir[File.dirname(__FILE__) + '/../config/initializers/*.rb'].sort.each{ |file| require file }
+
+require './util/error.rb'
+require './util/config.rb'
+Dir[File.dirname(__FILE__) + '/../models/*.rb'].sort.each{ |file| require file }
```

Comment:
It looks like these `Dir[...]` and `File.expand_path` can be simplified by `require_all` https://github.com/jarmo/require_all#usage

In fact this is the only functionality of `require_all` gem. We had a discussion before about this and we were both confused by `require_all` not requiring everything "automatically" but I just read their document and it doesn't do auto loading, all it does is to simplify writing the above complex require logic in a simpler fashion. 

(I noticed this a while back I didn't have time to fix these requires, do you want to attack it in this PR if not we can create a quality ticket?)

Reply:
I suggest to change that in another PR (issue to create), I've juste moved some code and add a line in this one

Reply:
this include is mandatory

Reply:
@senhalil you create the issue? I need to merge this PR

Reply:
https://gitlab.com/mapotempo/optimizer-api/-/issues/876